src/pretix/plugins/sendmail/views.py

```diff
@@ -20,6 +24,8 @@
 from pretix.plugins.sendmail.tasks import send_mails
 
 from . import forms
+from .models import Rule
+from ...control.views import CreateView, UpdateView
```

Comment:
Is there any reason not to import these from django.views.generic? Probably just a confused PyCharm